core/dbt/artifacts/resources/v1/metric.py

```diff
@@ -55,7 +63,7 @@ class MetricInput(dbtClassMixin):
     filter: Optional[WhereFilterIntersection] = None
     alias: Optional[str] = None
     offset_window: Optional[MetricTimeWindow] = None
-    offset_to_grain: Optional[TimeGranularity] = None
+    offset_to_grain: Optional[str] = None
```

Comment:
✅ This change of the type from `Optional[TimeGranularity]` to `Optional[str]` is safe and only a _minor_ artifact change, as the new type is a _superset_ of the old type.